Pages/Student/Create.cshtml

```diff
@@ -1,47 +1,91 @@
+                                          <span asp-validation-for="Input.Email" class="text-danger"></span>
+                                      </div>
+                                  </div>
+  
+                                  <div class="form-group row">
+                                      <label asp-for="Input.Mobile_Number" class="col-md-4 col-form-label text-md-right">Mobile_Number</label>
+                                      <div class="col-md-6">
+                                          <input asp-for="Input.Mobile_Number type="text" id="mobile_number" class="form-control">
+                                          <span asp-validation-for="Input.Mobile_Number" class="text-danger"></span>
+                                      </div>
+                                  </div>
+  
+                                  <div class="form-group row">
+                                      <label asp-for="Input.Address" class="col-md-4 col-form-label text-md-right">Address</label>
+                                      <div class="col-md-6">
+                                          <input asp-for="Input.Address" type="text" id="address" class="form-control">
+                                          <span asp-validation-for="Input.Address" class="text-danger"></span>
+                                      </div>
+                                  </div>
+  
+                                  <div class="form-group row">
+                                      <label asp-for="Input.BirthDate" class="col-md-4 col-form-label text-md-right">BirthDate</label>
+                                      <div class="col-md-6">
+                                          <input asp-for="Input.BirthDate"type="text" id="BirthDate" class="form-control" name="BirthDate">
+                                          <span asp-validation-for="Input.BirthDate" class="text-danger"></span>
+                                      </div>
+                                  </div>
+  
+                                  <div class="form-group row">
+                                      <label asp-for="Input.Year"" class="col-md-4 col-form-label text-md-right">Year</label>
+                                      <div class="col-md-6">
+                                          <input asp-for="Input.Year"type="text" id="year" class="form-control" name="year">
+                                          <span asp-validation-for="Input.Year" class="text-danger"></span>
+                                      </div>
+                                  </div>
+  
+                                      <div class="col-md-6 offset-md-4">
+                                          <button type="submit" class="btn btn-primary">
+                                          Submit
+                                          </button>
+                                      </div>
+                                  </div>
```

Comment:
This div is extra as it doesn't have closing tag. It is giving error. Delete it.